Sources/MiniPlayer/AudioMiniPlayer.swift

```diff
@@ -0,0 +1,168 @@
+/*****************************************************************************
+ * AudioMiniPlayer.swift
+ * VLC for iOS
+ *****************************************************************************
+ * Copyright (c) 2019 VideoLAN. All rights reserved.
+ * $Id$
+ *
+ * Authors: Soomin Lee <bubu # mikan.io>
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+@objc(VLCAudioMiniPlayer)
+class AudioMiniPlayer: UIView, MiniPlayer {
+    var visible: Bool = false
+    var contentHeight: Float {
+        return 72.0
+    }
+
+    @IBOutlet private weak var audioMiniPlayer: UIView!
+    @IBOutlet private weak var artworkImageView: UIImageView!
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var artistLabel: UILabel!
+    @IBOutlet private weak var progressBarView: UIProgressView!
+    @IBOutlet private weak var playPauseButton: UIButton!
+    @IBOutlet private weak var previousButton: UIButton!
+    @IBOutlet private weak var nextButton: UIButton!
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        initView()
+        setupConstraint()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func updatePlayPauseButton() {
+        let isPlaying = VLCPlaybackController.sharedInstance().isPlaying
+        playPauseButton.isSelected = !isPlaying
+    }
+}
+
+// MARK: - Private initializers
+
+private extension AudioMiniPlayer {
+    private func initView() {
+        Bundle.main.loadNibNamed("AudioMiniPlayer", owner: self, options: nil)
+        addSubview(audioMiniPlayer)
```

Comment:
This looks odd to me that the AudioMiniplayer loads a Nib that is also called AudioMiniPlayer and adds that as a subview. 
I would've expected this call to be wherever the AudioMiniplayer is instantiated and to have the rest of the setup code in initWithDecoder or awakeFromNib.